.github/workflows/main.yml

```diff
@@ -97,12 +97,10 @@ jobs:
         & compat\vcbuild\vcpkg_copy_dlls.bat release
         if (!$?) { exit(1) }
     - name: generate Visual Studio solution
-      shell: powershell
+      shell: cmd
       run: |
-        & .\git-sdk-64-minimal\usr\bin\bash.exe -lc @"
-          cmake ./contrib/buildsystems/ -DCMAKE_PREFIX_PATH=`pwd`/compat/vcbuild/vcpkg/installed/x64-windows -DMSGFMT_EXE=./git-sdk-64-minimal/mingw64/bin/msgfmt.exe -DPERL_TESTS=OFF -DPYTHON_TESTS=OFF -DCURL_NO_CURL_CMAKE=ON
-        "@
-        if (!$?) { exit(1) }
+        & 
+        cmake %PWD%/contrib/buildsystems/ -DCMAKE_PREFIX_PATH=%PWD%/compat/vcbuild/vcpkg/installed/x64-windows -DMSGFMT_EXE=%PWD%/git-sdk-64-minimal/mingw64/bin/msgfmt.exe -DPERL_TESTS=OFF -DPYTHON_TESTS=OFF -DCURL_NO_CURL_CMAKE=ON
```

Comment:
If you want to, you can also use `shell: bash`, which uses a Git for Windows Bash on the build agents.

Reply:
I having trouble specifying the absolute directory for CMAKE_PREFIX_PATH, will do

Reply:
I think it expands, but PWD is simply not a thing in `cmd.exe`.